### PR TITLE
Added formatter i I (w/ Deno.inspect)

### PR DESCRIPTION
**Pull request by imjamesb**
Added a formatter that uses Deno.inspect.

### CODE REVIEW

File: fmt/printf.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@
  * | `T`   | type of arg, as returned by `typeof`                           |
  * | `v`   | value of argument in 'default' format (see below)              |
  * | `j`   | argument as formatted by `JSON.stringify`                      |
+ * | `i I` | argument as formatted by `Deno.inspect`                        |
```
Comment (kt3k) [inline fmt/printf.ts:50]: Let's document the difference of `i` and `I`

```suggestion
 * | `i`   | argument as formatted by `Deno.inspect`                        |
 * | `I`   | argument as formatted by `Deno.inspect` in compact format      |
```

**Comment by kt3k** [comment]
I'm in favor of this addition. Does anyone have any concern on this?

**Comment by iuioiua** [comment]
LGTM

**Comment by lino-levan** [comment]
I see no reason not to. LGTM!

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**